src/store/ui/ui.ts

```diff
@@ -22,6 +22,8 @@ export const accountDrawerStep = atom<AccountStep>(AccountStep.Switch);
 // TODO: refactor into atom family
 export const showAccountsStore = atom(false);
 
+export const showHighFeeConfirmationState = atom(false);
```

Comment:
Could we display the high fee warning UI element without creating more global state? I know this is how it works atm, so can leave as is, but ideally we move away from this pattern, using either local state, or ideally, a route.

Reply:
Yeah, wasn't sure if I should follow this pattern! Haha, so hard when these patterns have been established. I question then, do I just do it here or refactor all of them?

Reply:
I am going to leave this for now but hope to revisit it when we refactor routing. We ran into an issue already with using routes for the`SignOutDrawer` and had to use global state in one place, so I think refactoring this pattern will be addressed by me next with routes and at that point we can remove `useDrawers` 🤞 . I would rather use routes here if can.